static/templates/actions_popover_content.handlebars

```diff
@@ -57,9 +57,10 @@
     {{/if}}
 
     <li>
-        <a href="{{ conversation_time_uri }}" class="popover_narrow_by_conversation_and_time" data-message-id="{{message.id}}">
-            <i class="icon-vector-link"></i> {{t "Link to this conversation" }}
+        <a href="#" class="copy_link" data-message-id="{{message.id}}">
+            <i class="icon-vector-link"></i> {{t "Copy link of message" }}
         </a>
+        <textarea class="copy_link_text" data-message-id="{{message.id}}">{{ conversation_time_uri }}</textarea>
```

Comment:
why do we need this hidden textarea?

We have `clipboard.js` in Zulip, probably we can use that to just set the clipboard content directly.